From fbb40cded429ad172671c5c6ca19e1ef4f00363e Mon Sep 17 00:00:00 2001
From: "an.xi" <an.xi@amlogic.com>
Date: Sat, 18 Sep 2021 11:11:53 +0800
Subject: [PATCH] SystemServer: add aml hal traces and complete the file [1/1]

PD#SWPL-58413

Problem:
The hal version of graphics is not in accord.

Solution:
Update the hal version of compower and allocator.

Verify:
Ohm

Change-Id: I5b28ff56649841cb37b4496c4edf159308b20e46
Signed-off-by: an.xi <an.xi@amlogic.com>
---
 services/core/java/com/android/server/Watchdog.java | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/services/core/java/com/android/server/Watchdog.java b/services/core/java/com/android/server/Watchdog.java
index 1cbf42ddf03a..565c3235c54d 100644
--- a/services/core/java/com/android/server/Watchdog.java
+++ b/services/core/java/com/android/server/Watchdog.java
@@ -110,8 +110,8 @@ public class Watchdog extends Thread {
         "/vendor/bin/hdmicecd",
         "/vendor/bin/hw/android.hardware.tv.cec@1.0-service",
         "/vendor/bin/hw/android.hardware.tv.input@1.0-service",
-        "/vendor/bin/hw/android.hardware.graphics.composer@2.3-service.droidlogic",
-        "/vendor/bin/hw/android.hardware.graphics.allocator@3.0-service",
+        "/vendor/bin/hw/android.hardware.graphics.composer@2.4-service.droidlogic",
+        "/vendor/bin/hw/android.hardware.graphics.allocator@4.0-service",
     };
 
     public static final List<String> HAL_INTERFACES_OF_INTEREST = Arrays.asList(
@@ -124,8 +124,8 @@ public class Watchdog extends Thread {
             "android.hardware.bluetooth@1.0::IBluetoothHci",
             "android.hardware.camera.provider@2.4::ICameraProvider",
             "android.hardware.gnss@1.0::IGnss",
-            "android.hardware.graphics.allocator@2.0::IAllocator",
-            "android.hardware.graphics.composer@2.1::IComposer",
+            "android.hardware.graphics.allocator@4.0::IAllocator",
+            "android.hardware.graphics.composer@2.4::IComposer",
             "android.hardware.health@2.0::IHealth",
             "android.hardware.light@2.0::ILight",
             "android.hardware.media.c2@1.0::IComponentStore",
-- 
2.26.1

